arduino-ide-extension/src/browser/contributions/survey-notification.ts

```diff
@@ -0,0 +1,80 @@
+import { MessageService } from '@theia/core';
+import { FrontendApplicationContribution } from '@theia/core/lib/browser';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { LocalStorageService } from '@theia/core/lib/browser';
+import { nls } from '@theia/core/lib/common';
+import { WindowService } from '@theia/core/lib/browser/window/window-service';
+import { ArduinoPreferences } from '../arduino-preferences';
+
+const SURVEY_MESSAGE = nls.localize(
+  'arduino/survey/surveyMessage',
+  'Please help us improve by answering this super short survey. We value our community and would like to get to know our supporters a little better.'
+);
+const DO_NOT_SHOW_AGAIN = nls.localize(
+  'arduino/survey/dismissSurvey',
+  "DON'T SHOW ANYMORE"
+);
+const GO_TO_SURVEY = nls.localize(
+  'arduino/survey/answerSurvey',
+  'ANSWER SURVEY'
```

Comment:
```suggestion
  "Don't show again"
);
const GO_TO_SURVEY = nls.localize(
  'arduino/survey/answerSurvey',
  'Answer survey'
```
The decision was made by Tooling Team design staff that caps should not be used in the Arduino IDE UI text.

Reply:
Thanks for the review. I fixed this [here](https://github.com/arduino/arduino-ide/pull/1035/commits/ae828a5c102daedaa5ae31c58b76b1ba5fb04167).